src/apps/ENGINE/Main.cpp

```diff
@@ -3,13 +3,16 @@
 #include "externs.h"
 #include "fs.h"
 #include "s_debug.h"
+#include "compiler.h"
 #include "steam_api.h"
 #include <crtdbg.h>
 #include <dbghelp.h>
 #include <spdlog/sinks/basic_file_sink.h>
 #include <stdio.h>
 #include <tchar.h>
 
+#define DUMP_FILENAME "engine_dump.dmp"
```

Comment:
I'd suggest avoiding using macros if possible. constexpr variable is better.

Reply:
OK, I'll do that.

I'm also changing #ifndef … #define … #endif checks to #pragma once, although #pragma is, in theory, outside of standard. However, AFAIK, all modern compilers properly supports it.

Reply:
This is to be defined in the coding style guidelines for the project which are not exist yet
Personally, I prefer #pragma once

Reply:
Strange, I made changes, I checked the code compiles, then opened GitHub Desktop – it reported about to different files, so I put it with the name “Update 1”, but I don't see that here…
Still have to learn much about that system. :(
PS. It seems to be here: https://github.com/storm-devs/storm-engine/commit/c99883c890dc968b36823ebd7b4df95807ff95ad
But why didn't it update it here?

Reply:
This diff is outdated, you commit is present in the new diffs

Reply:
> This diff is outdated, you commit is present in the new diffs

Thank you, now I see it's present in Commits tab.